elements/rh-footer/rh-footer.css

```diff
@@ -206,6 +206,10 @@ pfe-accordion {
   display: contents;
 }
 
+.isMobile .links {
+  --rh-footer-link-header-font-size: var(--rh-footer-link-header-mobile-font-size, var(--rh-font-size-body-text-lg, 1.125rem));
```

Comment:
Can you add this to the docblock?

Reply:
Yeah, I'm still deciding on whether to add it or not. I'd prefer to solve this will a part instead of a state-specific variable.

Reply:
agreed

Reply:
I added new parts for accordion-header and accordion-panel.  I like that much better :)